packages/patternfly-4/react-table/src/components/Table/Table.md

```diff
@@ -40,125 +40,179 @@ import {
+
+function SimpleCustomTable() {
+  const cells = [
+    // Equivalent to just passing the string
+    { title: 'Repositories' }, 
+    'Branches',
+    { title: 'Pull requests' },
+    'Workspaces',
+    // Will run the `transform` functions on the header, and the 
+    // `cellTransforms` on the cells. Transformers are used to inject
+    // extra props to the the cell component. In this case we use the
+    // builtin `textCenter` transformer to instruct the cell to apply
+    // the required stylings to center the cell content.
+    {
+      title: 'Last Commit',
+      transforms: [textCenter],
+      cellTransforms: [textCenter]
+    }
+  ];
+  const rows = [
+    ['Foo', 2, 0, 6, 1533635470],
+    [
+      // Cell content can also be defined as a JSX element.
+      // Extra props can be passed to the cell (the `td` element).
+      {
+        title: <div>Bar 👾</div>,
+        props: { title: 'hover title', colSpan: 3 }
+      },
+      2,
+      {
+        title: <div>1564566670</div>
+      }
+    ],
+    [
+      'Baz', 
+      6, 
+      4, 
+      99,
+      // In this example, we disable the default cellTransform for this specific cell, setting the `textCenter` prop to `false`.
+      {
+        title: '1565167870 (not centered)',
+        props: { textCenter: false }
+      }
+    ]
+  ];
+
+  return (
+    <Table caption="Cell/rows options" cells={cells} rows={rows}>
+      <TableHeader />
+      <TableBody />
+    </Table>
+  );
 }
 ```
 
-## Sortable table
+## Cell formatters
 
 ```js
 import React from 'react';
 import {
   Table,
   TableHeader,
   TableBody,
-  sortable,
-  SortByDirection,
-  headerCol,
-  TableVariant,
-  expandable,
-  cellWidth
+  textCenter
 } from '@patternfly/react-table';
+import {
+  CheckIcon,
+  TimesIcon
+} from '@patternfly/react-icons';
 
-class SortableTable extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      columns: [
-        { title: 'Repositories', transforms: [sortable] },
-        'Branches',
-        { title: 'Pull requests', transforms: [sortable] },
-        'Workspaces',
-        'Last Commit'
-      ],
-      rows: [['one', 'two', 'a', 'four', 'five'], ['a', 'two', 'k', 'four', 'five'], ['p', 'two', 'b', 'four', 'five']],
-      sortBy: {}
+function CellFormatters() {
+  const rows = [
+    ['Foo', 1533635470, 1],
+    ['Bar', 1564566670, 0],
+    ['Baz', 1565167870, 1],    
+  ];
+
+  // A component that displays a CI build status.
+  // *Heads-up* - this component definition should *not* stay inside
+  // the main component function in a real world application; it's
+  // done this way in this example as a work-around against the 
+  // limitation of one component per example of the documentation
+  // system in use.
+  const CIStatusIcon = ({ passing }) => {
+    const styles = {
+      color: passing 
+        ? 'var(--pf-global--success-color--200)' 
+        : 'var(--pf-global--danger-color--100)'
     };
```

Comment:
I am all for showing examples of custom CSS applications if they are useful and match PF standards. I don't know if we want a single "style customized" example or many, but I like this. "Cell Formatters" example is a very welcome addition. We could potentially have other examples of this coming in the future too (i.e. a table that has "favorited rows") cc: @patternfly/patternfly-core-ux

Reply:
I agree that these examples might be useful, maybe a section of the docs page on "customizing styles" with a few examples would be good in the long term, but we could start with just one.

As far as using the `--pf-global` variable directly as a string here, @priley86 do you happen to know if there is a way to import these color variables as constants from `@patternfly/react-styles` somehow?

Reply:
Love the "favorited rows" idea as an example.

@mturley yep, we could do something like `import { global_breakpoint_lg } from '@patternfly/react-tokens';` which basically results to 

```
{
    "name": "--pf-global--breakpoint--lg";
    "value": "992px";
    "var": "var(--pf-global--breakpoint--lg)";
}
```